typhos/suite.py

```diff
@@ -328,6 +334,20 @@ def tools(self):
                     for param in self.top_level_groups['Tools'].childs]
         return []
 
+    def _update_title(self, device=None):
+        """
+        Update the window title, optionally with a device.
+
+        Parameters
+        ----------
+        device : ophyd.Device, optional
+            Device to indicate in the title.
+        """
+        title_fmt = (self.DEFAULT_TITLE if device is None
+                     else self.DEFAULT_TITLE_DEVICE)
+
+        self.setWindowTitle(title_fmt.format(self=self, device=device))
```

Comment:
I notice we're passing in `self` to the formatter. Is there interesting information that we have in `TyphosSuite` that could be used in a title?

Reply:
Perhaps there is (or will be); not sure.